cylinder3d_node:
  ros__parameters:
    lidar_topic: "/carla/hero/lidar"
    frame_id: "lidar_frame"
    model_cfg: "/home/shareef/CARLA_PROJECT/Detection_system_carla_ros2/checkpoints/cylinder3d/semantickitti.yaml"
    model_ckpt: "/home/shareef/CARLA_PROJECT/Detection_system_carla_ros2/checkpoints/cylinder3d/model_save_backup.pt"
    use_gpu: true
